sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/LogRecordExporterConfiguration.java

```diff
@@ -91,6 +92,12 @@ static LogRecordExporter configureExporter(
             "Logging Log Exporter",
             "opentelemetry-exporter-logging");
         return SystemOutLogRecordExporter.create();
+      case "logging-otlp":
+        ClasspathUtil.checkClassExists(
+            "io.opentelemetry.exporter.logging.otlp.OtlpJsonLoggingLogRecordExporter",
+            "OTLP JSON Logging Log Exporter",
+            "opentelemetry-exporter-logging-otlp");
+        return OtlpJsonLoggingLogRecordExporter.create();
```

Comment:
WDYT about providing a `ConfigurableLogRecordExporterProvider` and letting the autoconfigure discover the exporter automatically?

Reply:
I could give a shot if it's desired.  Initially I was just following the code pattern used for the "logging" exporter.

Reply:
I've been thinking this may be a good idea to do for all the exporters from a separation of concerns perspective. If each exporter artifact contains a `Configurable{Signal}ExporterProvider`, then the code to interpret `ConfigProperties` becomes distributed into each artifacts, reducing the complexity of autoconfigure, especially around testing which has become cumbersome.

What do you think @jkwatson?

BTW, for the purposes of this PR I do think we should continue the current pattern. If we agree on implementing providers, we can open a separate issue to track that work.

Reply:
> then the code to interpret ConfigProperties becomes distributed into each artifacts, reducing the complexity of autoconfigure, especially around testing which has become cumbersome.

this sounds great, and could help guide users into good practices when they are trying to extend autoconfigure with their own items (e.g. exporters)

Reply:
> BTW, for the purposes of this PR I do think we should continue the current pattern. If we agree on implementing providers, we can open a separate issue to track that work.

👍